BUILD.tools

```diff
@@ -0,0 +1,30 @@
+# JVM artifacts that Pants itself depends on.
+# http://pantsbuild.github.io/dev_tasks.html#jvm-tool-bootstrapping
+
+# common rev for all org.scala-lang%* artifacts
+SCALA_REV = '2.10.5'
```

Comment:
To be more helpful, ScalaPlatform is in the mix when you use the jvm backend and that requires specs.